lib/parser.js

```diff
@@ -38,15 +33,18 @@ const readParcel = (
 };
 
 const parcel = ({
-// Encode parcel structure to Buffer
-  parcelId,     // id, // parcelId: 4b
-  parcelType,   // type, // parcelType: 1b
-  compression,  // compression, // 1b: no = 0, gzip = 1
-  encoding,     // encoding,
+  // Encode parcel structure to Buffer
+  parcelId, // number, id of parcel: 4b
+  parcelType, // number, type of parcel
+  // 1b: ping = 0, pong = 1, call = 2, callback = 3, event = 4, stream = 5
```

Comment:
Same here

Reply:
size of `parcelType` is 1 byte.
if `parcelType`  equal to 0 this means that this is a parcel of type `ping` etc.